src/Git/Commit.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace F3\Changelog\Git;
+
+class Commit
+{
+    /**
+     * @var string
+     */
+    private $hash;
+
+    /**
+     * @var \DateTIme
+     */
+    private $date;
+
+    /**
+     * @var string
+     */
+    private $subject;
+
+    public static function fromLogEntry(array $logEntry): self
```

Comment:
The constructor is not defined, making it possible to create an invalid (incomplete) instance of the class. Is that on purpose?